README.md

```diff
@@ -75,6 +75,7 @@ Following is a table of supported environment properties that can be used when r
 | :--------------- | :---------------------------------------------- | :-------------------------------------------------------------------------------------------- |
 | `HAL_IMAGE`      | `quay.io/halconsole/hal-development:latest`     | [HAL standalone image](https://hal.github.io/documentation/get-started/#container) to be used |
 | `WILDFLY_IMAGE`  | `quay.io/halconsole/wildfly-development:latest` | WildFly/JBoss EAP image to be used                                                            |
+| `SSO_IMAGE`      | `quay.io/keycloak/keycloak:latest`              | Keyclaok image to be used as SSO                                                              |
```

Comment:
> Keyclaok

-> Keycloak

Reply:
> Keyclaok image to be used as SSO

Keycloak image to be used for OIDC tests

Reply:
updated to `Keycloak/RH-SSO image to be used for OIDC tests`